monai/transforms/utility/array.py

```diff
@@ -415,8 +415,8 @@ class ToTensor(Transform):
         device: target device to put the converted Tensor data.
         wrap_sequence: if `False`, then lists will recursively call this function, default to `True`.
             E.g., if `False`, `[1, 2]` -> `[tensor(1), tensor(2)]`, if `True`, then `[1, 2]` -> `tensor([1, 2])`.
-        track_meta: whether to convert to `MetaTensor`, default to `False`, output type will be `torch.Tensor`.
-            if `None`, use the return value of ``get_track_meta``.
+        track_meta: whether to convert to `MetaTensor` or regular tensor, default to `None`,
```

Comment:
I would clarify the description a bit: "track_meta: if True convert to `MetaTensor`, otherwise to Pytorch Tensor, if None behave according to return value of `get_track_meta`."